packages/gasket-plugin-elastic-apm/lib/actions.js

```diff
@@ -0,0 +1,21 @@
+/** @type {import('@gasket/core').HookHandler<'actions'>} */
+module.exports = function actions(gasket) {
+  return {
+    async getApmTransaction(req) {
```

Comment:
I see there are some cases where the apm transaction will not be returned. I would think that this action would always return the apm transaction when you call it. I'm not at all familiar with the api though. Is it possible to return the apm transaction event if it has already started?

If not maybe a name like `startApmTransation ` could make more sense for this action?